resources/script/ace-builds/demo/bookmarklet/index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
```

Comment:
Hi,
I only need couple of files. I thought only the ones referenced in the html will be downloaded. Will update.

Thanks,
Ady

Sent from my iPhone

> On Sep 22, 2015, at 11:57 AM, Daniel Hindi <notifications@github.com> wrote:
> 
> In resources/script/ace-builds/demo/bookmarklet/index.html:
> 
> > @@ -0,0 +1,112 @@
> > +<!DOCTYPE html>
> do you really need to check in all the project? remember this will download to the phone potentially
> 
> —
> Reply to this email directly or view it on GitHub.
>